### PR TITLE
Allow $scvmm_log to be ignored by Rubocop

### PR DESCRIPTION
**Pull request by juliancheal**

### CODE REVIEW

File: .rubocop_local.yml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # GlobalVars:
 #   AllowedVariables:
```
Comment (durandom) [inline .rubocop_local.yml]: You'll have to uncomment these. rubocop_local

Comment (juliancheal) [inline .rubocop_local.yml]: Doh!

**Comment by juliancheal** [comment]
Thanks @durandom all working now. 🍏

**Comment by bronaghs** [comment]
@miq-bot add_label enhancement

**Pull request merged**